### PR TITLE
[ERE-1838] Initial RTL language support

### PR DESCRIPTION
**Pull request by ScriptSmith**

### CODE REVIEW

**Comment by ScriptSmith** [comment]
Was hoping I'd be able to add a new language with code `en-RTL` or similar, but ran into issues because

1. Django doesn't like adding new languages: https://stackoverflow.com/questions/12946830/how-to-add-new-languages-into-django-my-language-uyghur-or-uighur-is-not-su
2. `get_langauge_bidi` only cares about the direction of the  base language: https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/utils/translation/trans_real.py#L343

We could add the new language if required, but it works fine if you pick Arabic or another RTL language.

In your `.env_local`:

```bash
LANGUAGES="en ar"
```

File: rdrf/explorer/templates/explorer/base.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <script type="text/javascript" src="{% static 'js/vendor/underscore-1.13.1.min.js' %}"></script>
         
         <link rel="stylesheet" href="{% static 'css/jquery-1.13.0/themes/smoothness/jquery-ui.min.css' %}">
-        <link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.min.css' %}">
+        {% if LANGUAGE_BIDI %}<link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.rtl.min.css' %}">{% else %}<link rel="stylesheet" href="{% static 'vendor/bootstrap-5.1.0/css/bootstrap.min.css' %}">{% endif %}
```
Comment (ppettitau) [inline rdrf/explorer/templates/explorer/base.html]: Might be nicer split over multiple lines. 
Could we minimise the duplication by just replacing the `bootstrap.min.css` with the required resource?
Is there a better DRY way to output the bootstrap stylesheet includes across the many templates we declare them for?

**Review by ppettitau — CHANGES_REQUESTED** [review]
This has highlighted just how many "base" templates we have in TRRF.
It's not really the concern of this feature, but it might be nice to work towards having a true (and simple) base template that everything inherits from to keep things a bit cleaner.

File: rdrf/explorer/templates/explorer/base.html (hunk 1 above)
Comment (ScriptSmith) [reply rdrf/explorer/templates/explorer/base.html]: Yeah, I'll give it a go. I think the rtl stylesheet is different than the ltr one, but will double check.

Comment (ScriptSmith) [reply rdrf/explorer/templates/explorer/base.html]: Simplified the condition a bit. Other than that, we could create a custom template tag.

Comment (ppettitau) [reply rdrf/explorer/templates/explorer/base.html]: Looks better 👍

**Review by ppettitau — APPROVED** [review]

**Pull request merged**